scripts/update-catalog.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+This script adds a new bundle to the catalog.
```

Comment:
could we document that it updates the catalog template as well?

Reply:
done